lib/skunk/commands/default.rb

```diff
@@ -39,6 +37,8 @@ def execute
       #
       # @param [RubyCritic::AnalysedModulesCollection] A collection of analysed modules
       def report(analysed_modules)
+        Reporter.generate_report(analysed_modules)
```

Comment:
This was the magic line that allowed us to generate the json file in runtime (this is also how RubyCritic works).

In the future, I see Skunk following RubyCritic conventions and extracting the logic on `lib/skunk/commands/status_reporter.rb` to somewhere else and reusing it in the different generators.